api/user_setting.go

```diff
@@ -36,7 +36,7 @@ func (key UserSettingKey) String() string {
 }
 
 var (
-	UserSettingLocaleValue             = []string{"en", "zh", "vi", "fr", "nl", "sv", "de", "es", "uk", "ru", "it", "hant", "ko"}
+	UserSettingLocaleValue             = []string{"en", "zh", "vi", "fr", "nl", "sv", "de", "es", "uk", "ru", "it", "hant", "tr", "ko"}
```

Comment:
Can we have this order match the order in `web/src/components/LocaleSelect.tsx`

Reply:
In the other files `tr` is after `ko` lets keep it the same way here as well

Reply:
Updated. thx.